start_swarm_node.sh

```diff
@@ -7,7 +7,7 @@ docker run $@ \
   --volume /tmp/swarm:/swarm \
   --interactive \
   --tty \
-  "ethersphere/swarm:0.5.0" \
+  "ethersphere/swarm:0.5.2" \
```

Comment:
If you update the Swarm version here please also do it [here](https://github.com/MainframeHQ/erebos/blob/master/.travis.yml#L23) so the macOS and Linux versions are the same in CI.

Reply:
Hmm would love to, but I am not sure where to find the name of the new build?The current version has some sufix `c1c233d1` so I don't think that only version bump will work. Or will it?

Reply:
The version must match the one from the relevant [download link](https://swarm.ethereum.org/downloads/), so for 0.5.2 it should be `swarm-darwin-amd64-0.5.2-d8b56ff0`

Reply:
Thanks I missed that page.